### ISSUE TITLE
NTR: Monomethyl sterols high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8392]
New term request Monomethyl sterols high in blood (elthat_160318115411):

**Comment by pnrobinson** [comment]
@MickeySegal
See #8393 are you referring to 4-methylsterols?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000753
- label: Elevated circulating monomethyl sterol concentration
- parents: [Abnormal circulating lipid concentration (HP:0003119)](https://hpo.jax.org/app/browse/term/HP:0003119)
- pmid: PMID:21285510
- def: Concentration of monomethylsterols (such as 4alpha-monomethylsterol) in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8392
- synonyms:
